sr_description/hand/xacro/finger/lfmetacarpal/lfmetacarpal.urdf.xacro

```diff
@@ -60,8 +61,8 @@
     <joint name="${prefix}LFJ5" type="revolute">
       <parent link="${parent}" />
       <child link="${prefix}lfmetacarpal" />
-      <origin xyz="${reflect*-0.033} 0 0.02071" rpy="0 0 0" />
-      <axis xyz="0.573576436 0 ${reflect*0.819152044}" />
+      <origin xyz="${reflect*-0.033} 0 0.02071" rpy="0 ${reflect*-0.9599} 0" />
```

Comment:
can you show what does this change value apply to?

Reply:
It defines the pose of reference frame of (right/left hand) lfmetacarpal and also the axis of rotation of (right/left hand) LFJ5. You can check pictures 2 and 4 attached to this PR to inspect how lh_lfmetacarpal link of left hand changes

Reply:
@maxzieba merge if you're happy!